static/app/components/profiling/flamegraph/aggregateFlamegraph.tsx

```diff
@@ -218,7 +224,7 @@ export function AggregateFlamegraph(): ReactElement {
       const flamegraphFitTo = canvasHeight / flamegraph.depth;
       const minReadableRatio = 0.8; // this is quite small
       const fitToRatio = flamegraphFitTo / theme.SIZES.BAR_HEIGHT;
-      const barHeightRatio = Math.min(Math.max(minReadableRatio, fitToRatio), 1);
+      const barHeightRatio = Math.min(Math.max(minReadableRatio, fitToRatio), 1.2);
```

Comment:
i've allowed the bars to expand in instances where collapsing makes the flamegraph super tiny, or the flamegraph is small in general